rosdep/python.yaml

```diff
@@ -264,6 +264,10 @@ pynput-pip:
   ubuntu:
     pip:
       packages: [pynput]
+pyosmium:
+  debian: [python-pyosmium]
+  fedora: [python3-pyosmium]
```

Comment:
Fedora has dropped nearly all Python 2 packages, but it looks like that's what you're going for here (both the debian and ubuntu rules are for Python 2).

I think it would be better to exclude Fedora rather than include a dependency here that likely isn't going to work in your package.

Reply:
No problem.

Reply:
Can I get your approval?